common/common-rest/src/main/java/io/servicecomb/common/rest/codec/param/BodyProcessorCreator.java

```diff
@@ -58,7 +58,8 @@ public Object getValue(RestServerRequest request) throws Exception {
 
                 String contentType = request.getContentType();
                 if (contentType != null && contentType.startsWith(MediaType.TEXT_PLAIN)) {
```

Comment:
You can get the encoding information from the content-type , just like this "Content-Type: text/html; charset=utf-8" . 
I don't think using the "UTF-8"  without checking the charset is good idea. We can use the default setting as UTF-8, but when user specify the charset, we need to take it in the consideration.